tests/acceptance/features/webUISharingExternal/federationSharing.feature

```diff
@@ -359,3 +359,10 @@ Feature: Federation Sharing - sharing with users on other cloud storages
     And the user opens the share dialog for file "textfile.txt" using the webUI
     Then remote user "user1" should be listed as "Editor" via "simple-folder" in the collaborators list on the webUI
     And user "User Two" should be listed as "Editor" via "sub-folder" in the collaborators list on the webUI
+
+    Scenario: shares folder appears only after reloading the page
+      When user "user1" from remote server shares "simple-folder" with user "user1" from local server
+      And user "user1" from server "LOCAL" has accepted the last pending share
```

Comment:
something like this, because this is a when step but looks like given step here, because it says the user has accepted. 
```suggestion
      And user "user1" from server "LOCAL" accepts the last pending share
```

Reply:
I fixed it in the latest commit. Thank you for pointing it out.